src/index.js

```diff
@@ -227,3 +227,12 @@ export UnorderedList from 'carbon-components-react/lib/components/UnorderedList'
 export UNSTABLE__Pagination, {
   PageSelector,
 } from './components/UNSTABLE__Pagination';
+
+export {
```

Comment:
To be safe, how do you feel about marking these as unstable until we work out the correct naming?